File: Data Structure/Lab 3/SumArr.c
// Write a program to calculate sum of numbers from m to n.

#include<stdio.h>
void main(){
    int n,arr[n],i=0;
    int sum = 0;

    printf("Enter size of an array(n) :");
    scanf("%d",&n);

    for(i = 0; i < n ; i++){
        printf("Enter element :");
        scanf("%d",&arr[i]);
    }

    for(i = 0; i < n ; i++){
        sum = sum + arr[i];
    }

    printf("Sum of array element is : %d ",sum);
}

File: Data Structure(Java)/Lab 2/SumArr.java
// Write a program to calculate sum of numbers from m to n.

import java.util.Scanner;
public class SumArr {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        System.out.print("Enter n :");
        int n = sc.nextInt();
        int[] arr = new int[n];
        int sum = 0;

        for(int i = 0; i < n; i++){
            System.out.println("Enter number at "+i);
            arr[i] = sc.nextInt();
        }

        for(int i = 0; i < n; i++){
            sum += arr[i];
        }

        System.out.println(sum);

        sc.close();
    }
}
